guide/14-deep-learning/How CycleGAN Works.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<blockquote>Each generator has a corresponding discriminator model. The first discriminator model (Discriminator-A) takes real images from Domain-A and generated images from Generator-A and predicts whether they are real or fake. The second discriminator model (Discriminator-B) takes real images from Domain-B and generated images from Generator-B and predicts whether they are real or fake. </blockquote><p>Each generator has a corresponding discriminator model. The discriminator model takes real images from Domain and generated images from Generator to predict whether they are real or fake.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>